crates/swc_fast_ts_strip/src/lib.rs

```diff
@@ -441,6 +442,59 @@ impl TsStrip {
             self.add_overwrite(span.lo, b';');
         }
     }
+
+    fn strip_class_modifier(&mut self, mut start_pos: BytePos, key_pos: BytePos) {
+        let mut index = self.get_next_token_index(start_pos);
+
+        while start_pos < key_pos {
+            let TokenAndSpan { token, span, .. } = &self.tokens[index];
+            start_pos = span.hi;
+            index += 1;
+
+            let next = &self.tokens[index];
+
+            if next.had_line_break {
+                return;
+            }
+
+            // see ts_next_token_can_follow_modifier
+            // class { public public() {} }
+            if !matches!(
+                next.token,
+                Token::LBracket
+                    | Token::LBrace
+                    | Token::BinOp(BinOpToken::Mul)
+                    | Token::DotDotDot
+                    | Token::Hash
+                    | Token::Word(_)
+                    | Token::Str { .. }
+                    | Token::Num { .. }
+                    | Token::BigInt { .. }
+            ) {
+                return;
+            }
+
+            match token {
+                Token::Word(Word::Ident(IdentLike::Known(KnownIdent::Static))) => {
+                    continue;
+                }
+                Token::Word(Word::Ident(IdentLike::Known(
+                    KnownIdent::Readonly
+                    | KnownIdent::Public
+                    | KnownIdent::Protected
+                    | KnownIdent::Private,
+                ))) => {
+                    self.add_replacement(*span);
+                }
+                Token::Word(Word::Ident(IdentLike::Other(o))) if *o == "override" => {
```

Comment:
Adding `override` to the `KnownIdent` is likely a wise move.

Reply:
I think we should add it

Reply:
Let's add it to #9313.

Reply:
Sounds good 👍